pkgs/tools/misc/zsh-history-to-fish/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonApplication, fetchPypi, click }:
+
+buildPythonApplication rec {
+  pname = "zsh-history-to-fish";
+  version = "0.3.0";
```

```suggestion
  version = "0.3.0";
  format = "setuptools";
```
Please add `format`.